assets/js/plotter.js

```diff
@@ -363,4 +361,27 @@ function update() {
 	}]);
 	plot.setupGrid();
 	plot.draw();
-}
+} 
+// Pulls the calculated result from the state.dataset array if the user inputs are all numbers
+// @author Chris Want and Gabriella Wahl  
+function gen_output_value () {
+	variables = state.dataset
+	output_value = variables[0][1];
+	for (var i = 0; i < variables.length; i++) {
+		if(variables[i][1] !== variables[0][1]) { 
+			output_value = " "
```

Comment:
Would recommend trailing semicolons to mark the end of statements (otherwise javascript has to do more work to see if the next line is a continuation of the current line).

Reply:
Ok sounds great, I will make sure to continue that into the future edits!